quotas-and-limits_5cc850ba2c7d3a026fd42e7e.md

```diff
@@ -0,0 +1,81 @@
+Semaphore 2.0 enforces usage quotas and limits to protect customers and the
+platform from unforeseen spikes in usage. As your use of Semaphore expands over
+time, your quotas may increase accordingly.
+
+If you expect a notable upcoming increase in usage, you can proactively request
+quota adjustments by sending a request to <support@semaphoreci.com>.
+
+- [Quotas for Maximum Concurrent Running Jobs in an Organization](#quotas-for-maximum-concurrent-running-jobs-in-an-organization)
+- [Pipeline and Block Execution Time Limit](#pipeline-execution-time-limit)
+- [Job Log Output Size Limit](#job-log-output-size-limit)
+
+## Quotas for Maximum Concurrent Running Jobs in an Organization
```

Comment:
We don't use the word concurrent/concurrency anywhere on marketing site and app so I recommend saying "parallel" instead.